packages/reference/src/components/LinkActions/LinkEntityActions.tsx

```diff
@@ -90,32 +91,39 @@ export function useLinkActionsProps(props: LinkEntityActionsProps): LinkActionsP
     [sdk, entityType, onCreated]
   );
 
-  // Wrapping these two with useCallback caused a bug [ZEND-2154] where CTs were not propagated to the UI at all
```

Comment:
Doesn't the useCallback cause the issue any longer?

Reply:
Nope I checked. It does not